assets/eip-4675/contracts/NFT.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: ISC
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract NFT is ERC721, ERC721Enumerable, ERC721Burnable, Ownable {
+    constructor() ERC721("MyToken", "MTK") {}
+
+    function safeMint(address to, uint256 tokenId) public onlyOwner {
+        _safeMint(to, tokenId);
+    }
+
+    function _baseURI() internal pure override returns (string memory) {
+        return "Test";
+    }
+
+    function _beforeTokenTransfer(address from, address to, uint256 tokenId)
+    internal
+    override(ERC721, ERC721Enumerable)
+    {
```

Comment:
this function takes 4 arguments; but you have given three

Reply:
It could be three as well; check [here](https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20-_beforeTokenTransfer-address-address-uint256-)